zrml/authorized/src/mock_storage.rs

```diff
@@ -0,0 +1,52 @@
+// Copyright 2021-2022 Zeitgeist PM LLC.
```

Comment:
Shouldn't this be...?

```suggestion
// Copyright 2022 Forecasting Technologies LTD.
```